### PR TITLE
Kevin Song

### PR DESCRIPTION
**Pull request by ksong814**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

--------------------------

**WORKFLOW:**

To test if I had a successful connection from RStudio to my github repo, I first forked the hw01 repo and copied the url. I then created a new project (from RStudio) and then added in the Markdown content. Before I wrote my bio, I entered a few words and pushed the changes to ensure I had a successful connection. After that, I finished writing my bio with the Markdown, saved my work, tracked the differences, and committed the changes on the RStudio terminal console (I did so after configuring my username and email address). With integrated IDEs I notice that there are multiple ways to keep track of my work through git, whether that be through terminal commands from the console or more GUI approach the button options. With this assignment I used both methods, the commands through the terminal as well as through the IDE. With the terminal commands, I had to navigate between different directories to make sure I was in the **hw01** repo. Before that, I was in some git config folder and had to do a bunch of "cd .."s.

One issue that I had is that my picture was not uploading when I pushed it. I initially uploaded the picture to one of my other remote repos and copied the path name to add it as a domain, but I was receiving "alt text" rather than the image itself. To solve this issue, I cloned the hw01 repo on my local machine and pushed the picture remotely, just so that the image and README would be in the same directory. Then, I removed the domain name option in my image line (line 5), and just replaced it with the name of my image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Short Bio
+
+![alt text](meeeeee.jpg)
```
Comment (djanvier) [inline README.md:5]: Nice image! You can also write your code as `` if you're not intending to add a caption

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+![alt text](meeeeee.jpg) 
+
+Hi! My name is *Kevin Song* and I'm currently a second-year at the College. I grew up in New York City and went to school in Brooklyn. I plan to study both **Computer Science** and **Sociology** and hope to do research in computational social science. On campus, I am involved with UChicago TechTeam, a civic technology service group part of the IOP, and have a part-time job at TechBar. I love kimchi-fried rice, the *Sherlock* and *Elementary* series, and reasonably priced skincare.
+
+Some fun facts about me...
+* I am semi-related to a kpop star
+* Julianne Moore once gave me a hug somewhere in Manhattan
+* While living abroad, I made friends with many local shiba inus.
+
+I have recently gotten into programming as both an academic interest and creative hobby, here is a Python project where I tried to draw pixelated versions of my favorite characters: https://github.com/ksong814/Pixels (run main.py, I hope it works on your machine)
```
Comment (djanvier) [inline README.md:14]: The link can also be written as 

`[here is a Python project](https://github.com/ksong814/Pixels)`

Renders as:

 [here is a Python project](https://github.com/ksong814/Pixels)

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Reproducibility**         |      x     |              |            |

* Overall, good job!
* It sounds like uploading the image was a bit difficult. Both of your approaches were somewhat correct, but ultimately the easiest way to embed an image is to either upload it directly to your homework repo or to include it in the same folder as your README.md, although I would encourage using the latter approach since it gives you more practice pushing different files to GitHub

File: README.md (hunk 1 above)
Comment (ksong814) [reply README.md:5]: Thank you! And yeah I ended up pushing my image to the same directory as the README.

File: README.md [hunk 3]
```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Short Bio
+
+![alt text](meeeeee.jpg) 
+
+Hi! My name is *Kevin Song* and I'm currently a second-year at the College. I grew up in New York City and went to school in Brooklyn. I plan to study both **Computer Science** and **Sociology** and hope to do research in computational social science. On campus, I am involved with UChicago TechTeam, a civic technology service group part of the IOP, and have a part-time job at TechBar. I love kimchi-fried rice, the *Sherlock* and *Elementary* series, and reasonably priced skincare.
```
Comment (dasguptadinesh) [inline README.md:7]: Nice! I'm also a second-year as well.  Good use of italics and bold.

File: README.md (hunk 2 above)
Comment (dasguptadinesh) [reply README.md:14]: ^agreed.  Really neat that you can code these in Python though

Comment (dasguptadinesh) [reply README.md:14]: and good use of the bullet points.

**Pull request closed without merging**